src/openforms/js/components/admin/form_design/form-creation-form.js

```diff
@@ -392,15 +393,24 @@ function reducer(draft, action) {
          */
         case 'ADD_RULE': {
             const {form: {url}} = draft;
+            const emptyRule = action.payload;
             draft.logicRules.push({
-                ...EMPTY_RULE,
+                ...emptyRule,
```

Comment:
maybe we could even reduced the payload down to `isAdvanced` or not, and keep using the `EMPTY_RULE` constant + do the `isAdvanced` override in the reducer here. I think I'd prefer that - the `onAdd` then essentially just gains a single argument `isAdvanced`.

This keeps the state logic in the reducer.